packages/core/src/registries/designTokenRegistry.ts

```diff
@@ -3,13 +3,47 @@ import { builtInStyles, optionalBuiltInStyles } from '../definitions/styles';
 
 export let designTokensRegistry: DesignTokensDefinition = {};
 
+// This function is used to ensure that the composite values are valid since now those composite values are optional.
+// Therefore only border and in the future text related design tokens are/will be checked in this funciton.
+// Ensuring values for simplie key-value design tokens are not neccessary since they are required via typescript.
```

Comment:
"simple" instead of "simplie"?  I'm having a hard time reading/understanding this comment 🤔